tests/foreman/api/test_role.py

```diff
@@ -131,8 +180,16 @@ def test_positive_create_role_without_taxonomies(self):
 
         :CaseImportance: Critical
         """
+        role_name = gen_string('alpha')
+        role1 = entities.Role(
+            name=role_name,
+            organization=[],
+            location=[]
+        ).create()
+        self.assertEqual(role1.name, role_name)
+        self.assertEqual(role1.organization, [])
```

Comment:
nitpick: we usually prefer `self.assertFalse(role1.organization)`

Reply:
Well, I've never  noticed that before. But I think asserting against an empty list is more explicit. assertFalse will pass for [], 0, '', None and empty dict. So one would need to execute the code to really know what's the type of role1.organization. Besides that the name role1.organizagion is odd. If a list is expected shouldn't the attribute be called "organizations"?

Reply:
Well that's a point - to ensure there're no organizations, regardless of format returned - whether it's empty list, None, empty string or smth. It's less applicable for API tests since issues are rare due to nailgun nature, but is a common situation for cli tests - sometimes devs change output format few times in a row for different builds, once it's empty list, next time it's empty string, the next time it's `None` (not returned at all and None wasn't overwritten). After fixing such cases multiple times we've decided to use assertFalse - all that matters for us is a fact there's nothing assigned, output format isn't a big deal here.